src/packetq.cpp

```diff
@@ -69,6 +69,9 @@ static void usage(char* argv0, bool longversion)
         "  --csv | -c        CSV\n"
         "  --table | -t      Text table\n"
         "  --xml | -x ]      XML\n"
+        "  --rfc1035 | -e    Output DNS names escaped using RFC1035 format:\n"
+        "                    All characters outsize [a-zA-Z0-9_-] are escaped\n"
+        "                    like \\\\123. Beware of double-escaped \\.\n"
```

Comment:
Well, that's the reason for "beware" note. It is internally represeted as \123 so e.g. `WHERE qname = '\123'` works as you would expect, but it gets double-escaped when generating text output - so in the text output it is `\\123`.

I did not see a way how to avoid this without breaking backwards compatibility.

Reply:
If you're referring to CSV output that was changed in #92, it is different for different outputs.

The statement in this help text is wrong, as it's `\123` and not `\\123`.

Reply:
Maybe you should move the beware note to FAQ.md ?

Reply:
Okay, your change #92 fixes it so the warning is not true anymore and I removed it while removing -e short option.